addons/binding/org.openhab.binding.konnected/README.md

```diff
@@ -0,0 +1,90 @@
+# Konnected Binding
+
+This binding is for interacting with The Konnected Alarm Panel. A module which interfaces with existing home security sensors.  
+Konnected is an open-source firmware and software that runs on a NodeMCU ESP8266 device. 
+The Konnected hardware is specifically designed for an alarm panel installation, but the general purpose firmware/software can be run on any ESP8266 device.
+https://Konnected.io/_
+
+## Supported Things
+
+This binding supports one type of thing module, which represents a Konnected Alarm Panel.
+
+## Discovery
+
+The binding will auto discover The Konnected Alarm Panels which are attached to the same network as the server running openHAB via UPnP.  
+The binding will then create things for each module discovered which can be added.
+
+## Thing Configuration
+
+The binding attempts to discover The Konnected Alarm Panels via the UPnP service.
+The auto-discovery service of the binding will detect the ip address and port of the Konnected Alarm Panel.  
+But once it is added you will need to provide an Auth Token to secure communication between the module and openHAB.  
+As discussed above the binding will attempt to discover the ip address of your openHAB server.  However, if it is unable to determine the ip address you can manually define the ip address and port in the thing configuration.
+In addition you can also turn off discovery which when this setting is synced to the module will cause the device to no longer respond to UPnP requests as documented. https://help.konnected.io/support/solutions/articles/32000023968-disabling-device-discovery
+Please use this setting with caution and do not enable until a static ip address has been provided for your Konnected Alarm Panel via DHCP, router or otherwise.
+The blink setting will disable the transmission LED on the Konnected Alarm Panel.
+
+
+## Channels
+
+The auto discovered thing adds two default channels.
+
+| Channel  | Channel Id | Channel Type | Description |
+| ------------- | ------------- |-------------| -------------| 
+|1| Zone_6  | Switch  |A Switch channel for zone 6 |    
+|2|Zone_7  | Actuator  |The Channel for the Out Pin on the Konnected Alarm Panel|
```

Comment:
Is there a good reason to have this called "Zone_7" instead of e.g. "Out"?

Reply:
no.  It was a carry over from when i had to define the end number in the channel id.  That is no longer necessary and Out makes more sense logically with the physical device.